RELEASES.md

```diff
@@ -9,109 +9,102 @@
-| [v2.0.0](https://github.com/cosmos/interchain-security/releases/tag/v2.0.0) | 1.19 | v4.4.2 | v0.45.15-ics | v0.34.28 | 
-| [v2.1.0-provider-lsm](https://github.com/cosmos/interchain-security/releases/tag/v2.1.0-provider-lsm) | 1.19 | v4.4.2 | v0.45.16-ics-lsm | v0.34.28 | Provider only (Cosmos Hub specific) |
+| [v2.0.0](https://github.com/cosmos/interchain-security/releases/tag/v2.0.0) | 1.19 | v4.4.2 | v0.45.15-ics | v0.34.28 |
 | [v2.4.0-lsm](https://github.com/cosmos/interchain-security/releases/tag/v2.4.0-lsm) | 1.19 | v4.4.2 | v0.45.16-ics-lsm | v0.34.28 | Provider only (Cosmos Hub specific) |
 | [v3.1.0](https://github.com/cosmos/interchain-security/releases/tag/v3.1.0) | 1.20 | v7.1.0 | v0.47.3 | v0.37.2 |
-| v3.2.0 | 1.20 | v7.3.0 | v0.47.5 | v0.37.2 |
+| [v3.2.0](https://github.com/cosmos/interchain-security/releases/tag/v3.2.0) | 1.20 | v7.3.0 | v0.47.5 | v0.37.2 |
 | v3.3.0 | 1.20 | v7.3.0 | v0.47.5 | v0.37.2 |
 | v4.0.0 | 1.20 | v7.3.0 | v0.47.5 | v0.37.2 | Provider on >= v4.0.0 backwards compatible with consumers >= v3.2.0 |
 
 **Note:** For a list of major ICS features available in the currently active releases, see [FEATURES.md](./FEATURES.md).
 
 ### Backwards Compatibility
 
-A MAJOR version of ICS will always be backwards compatible with the previous MAJOR version of ICS. 
+A MAJOR version of ICS will always be backwards compatible with the previous MAJOR version of ICS.
 
 The following table indicates the compatibility of currently active releases:
 
-| Consumer | Provider | `v2.0.0` | `v2.1.0-provider-lsm` | `v2.4.0-lsm` | `v3.1.0` | `v3.2.0` | `v3.3.0` | `v4.0.0` | 
-|----------|----------|----------|-----------------------|--------------|----------|----------|----------|----------|
-| `v1.2.0-multiden` || ✅ (1) | ✅ (1) | ✅ (1) | ✅ (1),(2) | ✅ (1),(2) | ✅ (1),(2),(4) | ❌ |
-| `v2.0.0` || ✅ | ✅ | ✅ | ✅ (2) | ✅ (2) | ✅ (2),(4) | ❌ |
-| `v3.1.0` || ✅ | ✅ | ✅ | ✅ | ✅ | ✅ (4) | ❌ |
-| `v3.2.0` || ✅ | ✅ | ✅ | ✅ | ✅ | ✅ (4) | ✅ |
-| `v3.3.0` || ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| `v4.0.0` || ✅ (3) | ✅ (3) | ✅ (3) | ✅ (3) | ✅ | ✅ | ✅ |
+| Consumer | Provider | `v2.0.0` | `v2.4.0-lsm` | `v3.1.0` | `v3.2.0` | `v3.3.0` | `v4.0.0` |
+|----------|----------|----------|--------------|----------|----------|----------|----------|
+| `v2.0.0` || ✅ | ✅ | ✅ (1) | ✅ (1) | ✅ (1),(2) | ❌ |
+| `v3.1.0` || ✅ | ✅ | ✅ | ✅ | ✅ (2) | ❌ |
+| `v3.2.0` || ✅ | ✅ | ✅ | ✅ | ✅ (2) | ✅ |
+| `v3.3.0` || ✅ (2) | ✅ (2) | ✅ (2) | ✅ (2) | ✅ | ✅ |
+| `v4.0.0` || ✅ (2) | ✅ (2) | ✅ (2) | ✅ (2)| ✅ (2)| ✅ |
 
 #### Notes
 
-The following adjustments must be made to the CCV consumer genesis state that is obtained from the provider chain after the spawn time is reached in order for the consumer chain to start without errors. 
+The following adjustments must be made to the CCV consumer genesis state that is obtained from the provider chain after the spawn time is reached in order for the consumer chain to start without errors.
```

Comment:
Not specific to this PR but not clear to me the context of those `Notes` here. Are the notes intended for when someone is planning to use a previous MAJOR version of ICS or is it more general?

Reply:
The notes are intended for using two different versions of ICS on the consumer and provider.